tests/test_emitter.py

```diff
@@ -19,6 +19,7 @@
 import time
 import pytest
 import logging
+import typing as t
```

Comment:
Why not doing direct imports?
```suggestion
from typing import Type
```

Reply:
Sorry, that was a habit picked up from other projects. I'll adjust this.

Reply:
This is now adjusted to use a direct import.

Reply:
It seems not everywhere, right? :)